packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -189,6 +220,21 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
       );
     }
 
+    if (this.props.guardrailConfiguration !== undefined) {
```

Comment:
```suggestion
    if (!this.props.guardrailConfiguration) {
```
Cleaner

Reply:
Thank you. 
I think `!` is unnecessary, so I have removed it.